src/alexandria3k/data_sources/issn_subject_codes.py

```diff
@@ -102,11 +107,50 @@ def __init__(
             [issn_subject_codes_table],
             attach_databases,
         )
+        
+    def download_data(self, database_path):
```

Comment:
I guess the database path isn't needed, but a file name might, right?
You also need to provide a CLI way for calling this.

Reply:
I believe the database path is needed since the user must define it in order to get the ISSNs from the wanted works table at the database he needs.

Reply:
Ah, good point!  I now see there's an implicit dependency on the `works` table and some fields, which you should document.  Ideally, the SQL query should be specified by the user.  Please allow the specification of the file where the data are written.  Alternatively, you could populate the table on the fly, without an intermediate CSV file.